packages/core/ledger/ledger.go

```diff
@@ -86,8 +86,8 @@ func New(options ...Option) (ledger *Ledger) {
 	return ledger
 }
 
-// LoadOutputWithMetadatas loads OutputWithMetadatas from a snapshot file to the storage.
-func (l *Ledger) LoadOutputWithMetadatas(outputsWithMetadatas []*OutputWithMetadata) {
+// LoadFullUTXOStates loads OutputWithMetadatas from a snapshot file to the storage.
+func (l *Ledger) LoadFullUTXOStates(outputsWithMetadatas []*OutputWithMetadata) {
```

Comment:
I think the old name was more descriptive!

Reply:
That was my idea to change it to somenting that indicate more that this is snapshot state from before diffs and match more the FullIndex. Cause now it is not really obvious what are those indexes for